python/tank/platform/__init__.py

```diff
@@ -18,4 +18,5 @@
 from .engine import Engine
 from .framework import Framework
 from .util import change_context, get_framework, import_framework, current_bundle, restart, get_logger
+from .engine_events import *
```

Comment:
Agreed.  ```import *``` should be considered verboten.

PEP8 allows it to expose an internal interface publicly, but it introduces issues in knowing where a given identifier comes from and can accidentally expose internal code as part of the public interface (as engine_events evolves) so we shouldn't even do that.